angular/projects/spark-angular/src/lib/components/inputs/DatePicker.stories.ts

```diff
@@ -115,7 +115,7 @@ export const invalidDatePicker = () => ({
       </label>
       <sprk-icon
         iconType="calendar"
-        additionalClasses="sprk-c-Icon--stroke-current-color sprk-b-DatePicker__icon"
```

Comment:
If we remove `sprk-b-DatePicker__icon` from here, Is it being used anywhere else?

Reply:
It isn't. It didn't actually exist anywhere.